src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/db/Orderly.kt

```diff
@@ -93,7 +94,9 @@ class Orderly(val isReviewer: Boolean,
                     .join(ORDERLYWEB_PINNED_REPORT_GLOBAL)
                     .on(ORDERLYWEB_PINNED_REPORT_GLOBAL.REPORT.eq(REPORT_VERSION.REPORT))
                     .orderBy(ORDERLYWEB_PINNED_REPORT_GLOBAL.ORDERING)
-                    .fetchInto(ReportVersion::class.java)
+                    .fetch()
+
+            return mapToReportVersionsWithCustomFields(it, versions)
```

Comment:
The Pinned reports don't actually need the custom fields in the current front end, but we get them here anyway